app/envsettings.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+set -x
+
+export DEBUG=2
+export SECRET_KEY=foo
+export DJANGO_ALLOWED_HOSTS="localhost 127.0.0.1 [::1]"
+export DJANGO_SUPERUSER_USERNAME=admin
+export DJANGO_SUPERUSER_EMAIL=admin@admin.com
+export DJANGO_SUPERUSER_PASSWORD=admin
+# SQL_ENGINE=django.db.backends.postgresql
```

Comment:
Can commented lines be removed?

Reply:
I think we're just leaving this here to look at for the next PR.